app/controllers/admin/publishers/publisher_status_updates_controller.rb

```diff
@@ -20,6 +20,12 @@ def create
       end
     end
 
+    # If the user is being transitioned to only user funds remove all their promo registrations
+    if @publisher.only_user_funds?
+      @publisher.promo_registrations.destroy_all
```

Comment:
Hey @aekeus - curious how this works on the promo server side of things, if we mark a user as `no-ugp` on the promo side, and then try to update the `/api/1/promo/publishers/:ref_code` to be `"removed"` will this result in a 409?

Reply:
Yes likely, but I will need to test.

Reply:
No worries, we won't actually delete these based on the conversation in slack. Instead we'll just hide them in the view that way the reference to the promo server isn't destroyed